packages/common/src/templates/utils.ts

```diff
@@ -0,0 +1,45 @@
+import { IItem } from "@esri/arcgis-rest-types";
+import { dasherize } from "../utils";
+import { capitalize } from "../util";
+
+/**
+ * We do our best to glean what the solution template
+ * is supposed to become when activated. This util
+ * extracts the expected item type from the
+ * hubSolutionType typeKeyword on the solution item
+ *
+ * We use this in various UIs to show the expected
+ * type rather than "Solution"
+ *
+ * @param template
+ */
+export const getActivatedTemplateType = (template: IItem): string => {
```

Comment:
Wonder if we should call this `getDeployedTemplateType` ... or something using the term "deployed" vs "Activated", as it's more aligned with Solutions terminology (I believe... at least they add a Deployed keyword...)